### PR TITLE
Add WorkManager multiprocess sample

### PR DESCRIPTION
**Pull request by calren**

### CODE REVIEW

File: WorkManagerMultiprocessSample/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.google.android.material:material:1.3.0'
+    implementation 'androidx.concurrent:concurrent-futures-ktx:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'androidx.work:work-runtime-ktx:2.6.0-alpha02'
+    implementation 'androidx.work:work-multiprocess:2.6.0-alpha02'
+    testImplementation 'androidx.test:core:1.3.0'
+    testImplementation 'junit:junit:4.+'
```
Comment (keyboardsurfer) [inline WorkManagerMultiprocessSample/app/build.gradle:62]: Nit: Please don't use wildcards in dependency declarations.

File: WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/ExampleRemoteListenableWorker.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+package com.example.background.multiprocess
+
+import android.content.Context
+import android.util.Log
+import androidx.work.WorkerParameters
+import androidx.work.multiprocess.RemoteListenableWorker
+import com.google.common.util.concurrent.ListenableFuture
+import androidx.concurrent.futures.CallbackToFutureAdapter
+import kotlinx.coroutines.*
```
Comment (keyboardsurfer) [inline WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/ExampleRemoteListenableWorker.kt:25]: Nit: No wildcards here either. Studio tends to reset the default for wildcard imports.

File: WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/ExampleRemoteListenableWorker.kt [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+class ExampleRemoteListenableWorker(context: Context, parameters: WorkerParameters) :
```
Comment (keyboardsurfer) [inline WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/ExampleRemoteListenableWorker.kt:27]: Document what this worker does.

File: WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import androidx.work.ListenableWorker
+import androidx.work.OneTimeWorkRequest
+import androidx.work.WorkManager
+import androidx.work.multiprocess.RemoteCoroutineWorker
+import androidx.work.multiprocess.RemoteListenableWorker
+import androidx.work.multiprocess.RemoteListenableWorker.ARGUMENT_CLASS_NAME
+import androidx.work.multiprocess.RemoteListenableWorker.ARGUMENT_PACKAGE_NAME
+import androidx.work.multiprocess.RemoteWorkerService
+
+class MainActivity : AppCompatActivity() {
```
Comment (keyboardsurfer) [inline WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/MainActivity.kt:33]: Document the important bits to look at here, i.e. highlight `buildOneTimeWorkRemoteWorkRequest`.

**Review by keyboardsurfer — CHANGES_REQUESTED** [review]

**Comment by keyboardsurfer** [comment]
Maybe add a README as well.

File: WorkManagerMultiprocessSample/app/build.gradle (hunk 1 above)
Comment (calren) [reply WorkManagerMultiprocessSample/app/build.gradle:62]: Good call, done.

File: WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/ExampleRemoteListenableWorker.kt [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    RemoteListenableWorker(context, parameters) {
+
+    private var job: Job? = null
+
+    override fun startRemoteWork(): ListenableFuture<Result> {
```
Comment (tikurahul) [inline WorkManagerMultiprocessSample/app/src/main/java/com/example/background/multiprocess/ExampleRemoteListenableWorker.kt:32]: Consider using `doRemoteWork()`, the suspending method i.e. ? Unless the intent was to show a Java sample.

**Review by keyboardsurfer — APPROVED** [review]
Feel free to merge once @tikurahul's comments are addressed.

**Pull request merged**